### PR TITLE
remove comments

### PR DESCRIPTION
**Pull request by nuwantha-calcey**

### CODE REVIEW

File: EmployeeManagement/Controllers/DemoInternalEmployeesController.cs [hunk 1]
```diff
@@ -4,8 +4,6 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
-// labeler test
-// tettssdsa
```
Comment (nuwantha-calcey) [inline EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:8]: sadasdasdas

Comment (nuwantha-calcey) [reply EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:8]: sdfsdfsdf

Comment (nuwantha-calcey) [reply EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:8]: dasdasdasd

Comment (nuwantha-calcey) [reply EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:8]: dsadasdasd

Comment (nuwantha-calcey) [inline EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:8]: remove

File: .github/workflows/script.yml [hunk 2]
```diff
@@ -1,17 +1,35 @@
...
     runs-on: ubuntu-latest
     steps:
       - uses: actions/github-script@v6
         with:
           script: |
             github.rest.issues.addLabels({
               issue_number: context.issue.number,
-              owner: context.repository.owner,
-              repo: context.repository.repo,
+              owner: context.repo.owner,
```
Comment (NuwanthaSandakelum) [inline .github/workflows/script.yml:26]: can refactor

**Review by NuwanthaSandakelum — CHANGES_REQUESTED** [review]

File: .github/workflows/script.yml (hunk 2 above)
Comment (nuwantha-calcey) [reply .github/workflows/script.yml:26]: done

**Review by NuwanthaSandakelum — APPROVED** [review]
LGTM!

File: EmployeeManagement/Controllers/DemoInternalEmployeesController.cs [hunk 3]
```diff
@@ -4,8 +4,6 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
-// labeler test
```
Comment (NuwanthaSandakelum) [inline EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:7]: remove comment

**Review by NuwanthaSandakelum — CHANGES_REQUESTED** [review]

File: EmployeeManagement/Controllers/DemoInternalEmployeesController.cs (hunk 3 above)
Comment (nuwantha-calcey) [reply EmployeeManagement/Controllers/DemoInternalEmployeesController.cs:7]: dsadas

**Review by NuwanthaSandakelum — APPROVED** [review]
nice

**Review by NuwanthaSandakelum — APPROVED** [review]

**Review by NuwanthaSandakelum — CHANGES_REQUESTED** [review]
sdfsdf

**Review by NuwanthaSandakelum — APPROVED** [review]

**Pull request closed without merging**